dataprocessors/json/schema.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Probably ideal if we can bring this in from `spiceai/spiceai` instead of living in the data-components repo.